RuddockWebsite/modules/account/helpers.py

```diff
@@ -0,0 +1,79 @@
+from flask import g, flash
+from sqlalchemy import text
+from RuddockWebsite import auth_utils
+from RuddockWebsite import email_templates
+from RuddockWebsite import email_utils
+from RuddockWebsite import misc_utils
+from RuddockWebsite import validation_utils
+
+def get_user_data(user_id):
+  ''' Helper funciton to get user data. '''
+  query = text("""
+    SELECT fname, lname, uid, matriculate_year, grad_year, email
+    FROM members
+    WHERE user_id=:uid
+    """)
+  return g.db.execute(query, uid=user_id).first()
+
+def handle_create_account(user_id, username, password, password2, birthday):
+  '''
+  Creates a new account. Flashes a message and returns False if an error
+  occurs, otherwise returns True.
+  '''
+  # Validate username and password. The validate_* functions will flash errors.
+  # We want to check all fields and not just stop at the first error.
+  is_valid = True
+  if not validation_utils.validate_username(username):
+    is_valid = False
+  if not validation_utils.validate_password(password, password2):
+    is_valid = False
+  if not validation_utils.validate_birthday(birthday):
+    is_valid = False
+
+  if is_valid:
```

Comment:
invert this conditional and return `False` to remove one level of indentation from the following block